forms-flow-web/src/models/EmployeeDataModel.js

```diff
@@ -0,0 +1,56 @@
+export default class EmployeeDataModel {
+  // Use this class properties names in the form builder ("Property Name" under API tab) to fill the form with the employee data.
+
```

Comment:
I standardize the employeeData field names so don't be mixed with regular fields. If one wants a field to be populated with employeeData should use these names in the API tab of the form builder. For example, a first name filed should use `__firstName` as the property name.

Reply:
These are almost all of the data I get from the Telus endpoint with that test auth token. Not sure if we need all, so let me know if some need to be filtered.